### PR TITLE
feat: correct sensitive config, store keys encrypted

### PR DESCRIPTION
**Pull request by andrey18106**
Encrypt admin and user `api_key_v3` and `api_key_v4`

### CODE REVIEW

File: lib/Service/TmdbAPIService.php [hunk 1]
```diff
@@ -234,9 +238,10 @@ public function getImage(string $size, string $imagePath): array {
 	private function getApiKeyV3(?string $userId): string {
 		$adminApiKey = $this->config->getAppValue(Application::APP_ID, 'api_key_v3');
 		if ($userId === null) {
-			return $adminApiKey;
+			return $adminApiKey !== '' ? $this->crypto->decrypt($adminApiKey) : '';
 		}
-		return $this->config->getUserValue($userId, Application::APP_ID, 'api_key_v3', $adminApiKey) ?: $adminApiKey;
+		$userApiKey = $this->config->getUserValue($userId, Application::APP_ID, 'api_key_v3', $adminApiKey) ?: $adminApiKey;
+		return $userApiKey !== '' ? $this->crypto->decrypt($userApiKey): '';
```
Comment (julien-nc) [inline lib/Service/TmdbAPIService.php:244]: This is potentially decrypting the admin key twice (if there is no user key).

File: lib/Migration/Version2100Date202410231627.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		$clientId = $this->config->getAppValue(Application::APP_ID, 'api_key_v3');
+		if ($clientId !== '') {
+			$clientId = $this->crypto->encrypt($clientId);
+			$this->config->setAppValue(Application::APP_ID, 'api_key_v3', $clientId);
+		}
+		$clientSecret = $this->config->getAppValue(Application::APP_ID, 'api_key_v4');
+		if ($clientSecret !== '') {
+			$clientSecret = $this->crypto->encrypt($clientSecret);
+			$this->config->setAppValue(Application::APP_ID, 'api_key_v4', $clientSecret);
+		}
```
Comment (julien-nc) [inline lib/Migration/Version2100Date202410231627.php:48]: Wrong variable names :grin:

**Review by julien-nc — CHANGES_REQUESTED** [review]

File: lib/Service/TmdbAPIService.php (hunk 1 above)
Comment (andrey18106) [reply lib/Service/TmdbAPIService.php:244]: Why? Only the result of condition which is encrypted string yet from config would be decrypted.

Comment (julien-nc) [reply lib/Service/TmdbAPIService.php:244]: Nevermind, I've read too fast.

**Review by julien-nc — APPROVED** [review]

**Pull request merged**